### PR TITLE
Add starting date in the stock sheet report

### LINKED ISSUE #4401: Add a start date limit to fiche de stocks report
Here is the current presentation of the fiche de stocks:
image

It offers only an end date. The problem is that when you have a busy medicine (one with lots of entries and exits) the report gets really difficult to read.  Here is an example: 
KmjE7r65ZH

You can see how, as time goes on, this will get really difficult to manage.  I propose that we add a start date and end date so that we can produce more valuable reports.

### PR DESCRIPTION
**Pull request by mbayopanda**
image

image

Close #4401

### CODE REVIEW

File: client/src/modules/reports/generate/stock_sheet/stock_sheet.config.js [hunk 1]
```diff
@@ -12,6 +12,7 @@ function StockSheetConfigController($sce, Notify, SavedReports, AppCache, report
   const reportUrl = 'reports/stock/sheet';
 
   vm.previewGenerated = false;
+  vm.havingDateInterval = 0;
```
Comment (jmcameron) [inline client/src/modules/reports/generate/stock_sheet/stock_sheet.config.js]: I think a more natural variable name is:  vm.hasDateInterval

**Review by jmcameron — COMMENTED** [review]
It looks good to me (but I have not actually tried it).  I added one comment about the "having..." variable name.

**Review by jniles — COMMENTED** [review]
I agree with @jmcameron's comments.  Otherwise, LGTM.

**Comment by jniles** [comment]
LGTM

bors r+

**Pull request merged**